### PR TITLE
`0.12.0` change notes.

### PR DESCRIPTION
**Pull request by pq**
@devoncarew @skybrian @mit-mit

### CODE REVIEW

**Comment by devoncarew** [comment]
lgtm! I would also add support for 2017.1

**Comment by pq** [comment]
> lgtm! I would also add support for 2017.1

Added!

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -21,6 +21,17 @@
 
   <change-notes>
     <![CDATA[
+
+0.12.0:
```
Comment (skybrian) [inline resources/META-INF/plugin.xml]: I thought we were going to whole numbers?

Comment (devoncarew) [inline resources/META-INF/plugin.xml]: Yeah, +1. @mit-mit had some ideas around the versioning, but without other guidance let's go with `12.0`.

Comment (mit-mit) [inline resources/META-INF/plugin.xml]: Yes, I suggested we make this 1.0 (as in our first, complete and stable plugin). It seems little odd to me to start with 12.0.

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -21,6 +21,16 @@
...
+<ul>
+  <li>new Flutter `stless`, `stful`, and `stanim` live templates</li>
+  <li>support for specifying "Additional Args" to Flutter application launches</li>
+  <li>default run configuration creation on project open (when possible)</li>
+  <li>device menu improvements</li>
+  <li>miscellaneous bug fixes</li>
+</ul>
```
Comment (mit-mit) [inline resources/META-INF/plugin.xml]: Mention causal stack traces?

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: I'd rather not personally.  `1.0` suggests something more than we want to (I think).  If you guys feel strongly though I won't resist.

Comment (pq) [inline resources/META-INF/plugin.xml]: In particular, I don't know that this assessment

> as in our first, complete and stable plugin

is really accurate.

Again, I won't fight this if you all feel strongly.

File: resources/META-INF/plugin.xml (hunk 2 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: Good idea.  Can you think of a short sentence?

**Pull request merged**